packages/cli/test/scripts/unpack.test.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+require('../setup');
+
+import sinon from 'sinon';
+import fs from 'fs-extra';
+import cp from '../../src/utils/cp';
+import patch, { cache } from '../../src/utils/patch';
+import unpack from '../../src/scripts/unpack';
+
+const simpleGit = patch('simple-git/promise');
+
+const repo = 'zeppelinos/zepkit';
+const url = `https://github.com/zeppelinos/zepkit.git`;
+
+describe('unpack script', function() {
```

Comment:
We definitely need an integration test for running this command. Maybe we can tackle it along with the integration tests of the zepkit itself?

Reply:
Yeah. But I wouldn't do it as a part of `cli` because downloading and installing a kit few times will take like 10 minutes. We have to be smart about it. It should be a separate test suite for sure.

Reply:
Check out the `test` folder at the root of the project, there is already an integration test suite. We could add a second one there.